packages/opentelemetry-base/src/platform/browser/constants.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { VERSION } from '../../version';
+
+/** Constants describing the SDK in use */
+export const SDK_INFO = {
+  NAME: 'opentelemetry',
+  RUNTIME: 'browser',
+  LANGUAGE: 'webjs',
```

Comment:
language should be `javascript` for both no ?

Reply:
The Resource spec differentiates between `webjs` and `nodejs` depending on  runtime. See: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-resource-semantic-conventions.md#library.

Note, I am using the updated attribute names from: https://github.com/open-telemetry/opentelemetry-specification/pull/494, but the values are unchanged.